app/controlplane/plugins/core/guac/v1/README.md

```diff
@@ -0,0 +1,96 @@
+# GUAC integration
+
+Graph for Understanding Artifact Composition ([GUAC](https://github.com/guacsec/guac)) aggregates software security metadata into a high fidelity graph database—normalizing entity identities and mapping standard relationships between them. Querying this graph can drive higher-level organizational outcomes such as audit, policy, risk management, and even developer assistance.
+
+This integration allows Chainloop users to automatically send attestation (DSSE envelopes / in-toto statements) and CycloneDX/SPDX Software Bill Of Materials (SBOMs) to a cloud storage bucket staging area. From there, GUAC can be configured to continuously monitor and inject that data.
+
+![GUAC integration](./img/overview.png)
+
+Currently, the integration only supports [Google Cloud Storage](https://cloud.google.com/storage) (GCS) as a storage provider.
+
+## How to use it
+
+### Requirements
```

Comment:
- I would make requirements generic, "you need a writable cloud storage bucket"
- I would add another section: supported cloud storage platforms
- I would move GCP specific info (like service account, json creds file) to a separate GCP section

Reply:
Not sure, if feels like an early optimization. Note that right above it I explicitly mention that only GCP is supported. 

Let me think about it, it will definitely make sense if/when we add other backend providers.

Reply:
I just do not want people to assume that this is the default option and show that more storage options are coming in the future.